implementation/BeforeInstallPromptEvent.js

```diff
@@ -0,0 +1,143 @@
+/*globals BeforeInstallPromptEvent, DOMException*/
```

Comment:
Should this file be surrounded in a closure so it hides all the other variables?